init.c

```diff
@@ -2059,15 +2061,16 @@ static int __parse_jobs_ini(struct thread_data *td,
 
 		ret = fio_options_parse(td, opts, num_opts);
 
-		if (!ret) {
-			if (!strcmp(file, "-") && td->o.read_iolog_file != NULL) {
-				char *fname = get_name_by_idx(td->o.read_iolog_file,
-							      td->subjob_number);
-				if (!strcmp(fname, "-")) {
-					log_err("fio: we can't read both iolog "
-						"and job file from stdin.\n");
+		if (!ret && td->o.read_iolog_file != NULL) {
+			char *fname = get_name_by_idx(td->o.read_iolog_file,
+						      td->subjob_number);
+			if (!strcmp(fname, "-")) {
+				if (stdin_occupied) {
+					log_err("fio: we can't read multiple iologs "
```

Comment:
Maybe invert the sentence?

"fio: only one of read_iolog_file/job file can be read from stdin but more than one was found"